### ISSUE TITLE
Plugin failing with error: Invalid AppSync Configuration:      at appSync: must be object

### ISSUE THREAD

**Issue by sailleshp** [sid88in/serverless-appsync-plugin#566]
Hi Team,

We are intermittently getting below error from last 2 days due to which we are not able to deploy our binaries.
It was working fine last week.
Is there anything that you guys can help please.

![image](https://user-images.githubusercontent.com/69572750/220740773-60286728-98fb-4990-b890-2f764374a108.png)

**Comment by bboure** [comment]
Hi,

It seems to be a validation issue. 

Please note: 2 days ago, we release v2 of this plugin.
Which version are you using or trying to use? 

From the error "appSync: must be object", I suspect you're using the wrong version. 
If you are trying to use v1, please make sure you pinned the plugin version in your `package.json` file and that you haven't upgraded by mistake.

**Comment by sailleshp** [comment]
Thanks for replying @bboure .
We have identified the issue, in workflow file we were installing latest version of plugin which installed V2 version due to which it failed the entire deployment.
Though I tried to move to V2 by following below documentation https://github.com/sid88in/serverless-appsync-plugin/blob/master/doc/upgrading-from-v1.md which doesn't seem to be straight forward. But will try again and update here.
